components/x-live-blog-wrapper/src/LiveBlogWrapper.jsx

```diff
@@ -0,0 +1,55 @@
+import { h } from '@financial-times/x-engine';
+import { LiveBlogPost } from '@financial-times/x-live-blog-post';
+import { withActions } from '@financial-times/x-interaction';
+import { listenToLiveBlogEvents } from './LiveEventListener';
+
+const withLiveBlogWrapperActions = withActions({
+	insertPost (post) {
```

Comment:
When posts are updated and inserted, does the application take charge of informing the user about it?

Reply:
@nickcolley there is currently a notification that appears and takes you to the new post should you want to. This component is new so we are making some of these architectural decisions along the way and one that we have defined yet is how much of that should live in here and how much should live in the application.

The only version of these pages wasn't amazing for accessibility and I don't think it told you about updates or deletes and only mentioned creates. So we must improve this and I think it will be the main factor in "whose" responsibility that is, the app or the component. At this stage I am leaning towards component.

Reply:
Sorry that was a bit of a ramble to acknowledge that yes we need to make sure the user experience is accessible